metrics/serializers.py

```diff
@@ -0,0 +1,159 @@
+from rest_framework import serializers
+from .models import *
+
+
+class SensorTypeSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = SensorType
+        fields = ['id', 'name']
+
+
+class UnitSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Unit
+        fields = ['id', 'name']
+
+
+class SensorListSerializer(serializers.ModelSerializer):
+    """ GET --> just a list of all sensors"""
+
+    sensors_count = serializers.SerializerMethodField()
+
+    def get_sensors_count(self, obj):
+        return Sensor.objects.all().count()
+
+    class Meta:
+        model = Sensor
+        fields = [
+            # read_only_fields
+            'sensors_count',
+            'sensor_model',
+            'id',
+            'title',
+            'description',
+            'type',
+            'unit',
+        ]
+        read_only_fields = ['sensors_count',
+                            'sensor_model',
+                            'id',
+                            'title',
+                            'description',
+                            'type',
+                            'unit',
+                            ]
+
+
+class SensorRetrieveSerializer(serializers.ModelSerializer):
```

Comment:
```suggestion
class SensorDetailSerializer(serializers.ModelSerializer):
```
I hope this change means what I mean

Reply:
fixed